specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2022-08-11-preview/HybridCompute.json

```diff
@@ -0,0 +1,1627 @@
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.HybridCompute/locations/{location}/publishers/{publisher}/extensionTypes/{extensionType}/versions": {
+      "get": {
+        "tags": [
+          "extensions"
+        ],
+        "operationId": "ExtensionMetadata_List",
+        "description": "Gets all Extension versions based on location, publisher, extensionType",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The location of the Extension being received."
+          },
+          {
+            "name": "publisher",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The publisher of the Extension being received."
+          },
+          {
+            "name": "extensionType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The extensionType of the Extension being received."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ExtensionValueListResult"
+            }
+          },
+          "default": {
+            "description": "Error in retrieving extension list",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "GET a list of extensions": {
+            "$ref": "./examples/ExtensionMetadata_List.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.HybridCompute/operations": {
+      "get": {
+        "tags": [
+          "operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Gets a list of hybrid compute operations.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Comment:
You are using both v2 and v3 common-types. Could we change all these to v3? This should fix the issue in Go SDK genration

Reply:
Hi Dapeng - This change is outside of the scope of the new resource we are introducing. Will add the v3 change to the backlog as we have a few items from the review

Reply:
Workitem for the updates, for next iteration
https://msazure.visualstudio.com/defaultcollection/One/_workitems/edit/15685839

Reply:
Discussed with the author offline, we cannot make the change right now. I am going to add a deduplication flag in go configuration to unblock this temporarily.

Reply:
@ankjayant - pls incorporate this change in your branch:
https://github.com/ArcturusZhang/azure-rest-api-specs/commit/b3de42a2d05e009eaaef508ed47db1a7523852f4

This is for the failing Go pipeline

Reply:
> @ankjayant - pls incorporate this change in your branch: [ArcturusZhang@b3de42a](https://github.com/ArcturusZhang/azure-rest-api-specs/commit/b3de42a2d05e009eaaef508ed47db1a7523852f4)
> 
> This is for the failing Go pipeline

same issue for python